packages/domain-search/src/hooks/use-suggestions-list.ts

```diff
@@ -1,25 +1,48 @@
-import { type DomainAvailability, DomainAvailabilityStatus } from '@automattic/api-core';
+import {
+	type DomainAvailability,
+	DomainAvailabilityStatus,
+	fetchDomainAvailability,
+} from '@automattic/api-core';
 import { useQueries, useQuery, UseQueryResult } from '@tanstack/react-query';
-import { useCallback, useMemo } from 'react';
+import { useCallback, useEffect, useMemo } from 'react';
 import { getTld } from '../helpers';
 import { partitionSuggestions } from '../helpers/partition-suggestions';
 import { useDomainSearch } from '../page/context';
 
 export const useSuggestionsList = () => {
-	const { query, queries, config } = useDomainSearch();
+	const { query, queries, config, events } = useDomainSearch();
 
 	const { data: suggestions = [], isLoading: isLoadingSuggestions } = useQuery( {
 		...queries.domainSuggestions( query ),
 		enabled: true,
+		// We should just get suggestions when the query changes
+		staleTime: Infinity,
+		refetchOnMount: false,
+		refetchOnWindowFocus: false,
 	} );
 
+	useEffect( () => {
```

Comment:
Are we sure this won't trigger multiple times for the same query?

Reply:
At least in my tests this triggers only once, if the suggestions query also only runs once.